Userland/Services/WebServer/main.cpp

```diff
@@ -99,7 +104,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     out("Listening on ");
     out("\033]8;;http://{}:{}\033\\", ipv4_address.value(), server->local_port());
     out("{}:{}", ipv4_address.value(), server->local_port());
-    outln("\033]8;;\033\\");
+    out("\033]8;;\033\\");
+    outln("...");
```

Comment:
There is no purpose in adding "..." here. "Listening on http://0.0.0.0:8080" is a perfectly valid line in itself.